File: 0274-h-index/0274-h-index.java
class Solution {
    public int hIndex(int[] citations) {
        //3 0 6 1 5 
        //0 1 3 5 6 
        //1 1 3 
        Arrays.sort(citations); 
        int start = 0;  
        while(start < citations.length && citations[citations.length-1 - start] > start){
            start++; 
        }
        return start; 
    }
}

File: 0274-h-index/0274-h-index.cpp
class Solution {
public:
    int hIndex(vector<int>& citations) {
        sort(citations.begin(), citations.end());
        for(int i = 0; i < citations.size(); i++){
            int hIndex = citations.size() - i; 
            if(citations[i] >= hIndex) return hIndex; 
        }
        return 0;  
    }
};